### ISSUE TITLE
Error in French Confirmed Cases

### ISSUE THREAD

**Issue by GerardoRTobar** [CSSEGISandData/COVID-19#2203]
Has anyone noticed the error in France (the one with no Province/State entry), which for 13/4/2020 had 136,779 and for 14/4/2020 has 130,253.

This happens in the time_series_covid19_confirmed_global.csv file.

**Comment by GerardoRTobar** [comment]
I hope this gets fixed soon. I mean, as it is an acumulative variable, every day it should present at least equal or higher value, but not lest than the prior day.

**Comment by MartinFavier** [comment]
I agree with you, I have 6514 confirmed cases less than the prior day.
Does anyone know why ?

**Comment by GerardoRTobar** [comment]
It must be typing error. I hope they get this fixed. They made some commits fixing some stuff, but I think they either forgot about this or didn't spot the issue.

**Comment by MartinFavier** [comment]
OK, let's keep up to date if we find a data correction

**Comment by alfkoehn** [comment]
Correct numbers are given here: https://github.com/CSSEGISandData/COVID-19/issues/2202

**Comment by advaithasabnis** [comment]
#2188 
Should probably be +6514.

@alfkoehn I think France mentions cases from nursing homes separately so it's 103k + about 40k from there [[Source](https://www.santepubliquefrance.fr/maladies-et-traumatismes/maladies-et-infections-respiratoires/infection-a-coronavirus/articles/infection-au-nouveau-coronavirus-sars-cov-2-covid-19-france-et-monde)]? Not sure but almost everywhere (JHU, worldometer, etc.) I see the sum of these two numbers.

**Comment by alfkoehn** [comment]
> @alfkoehn Not sure but almost everywhere (JHU, worldometer, etc.) I see the sum of these two numbers.

True, but data maintainer of this website have themselves stated _"After careful consideration and extensive research, we cannot confirm beyond a reasonable doubt that these two categories are mutually exclusive."_ [1]

This means the numbers should still be treated separately in my understanding - and is also the conclusion of this database's maintainer. Anyway, I did not wanted to start this discussion here again, I just thought it would be helpful to give the (some?) "correct" number instead of just saying something is wrong.

[1] https://github.com/CSSEGISandData/COVID-19/issues/2094

**Comment by yetzt** [comment]
Thats what happens when you count speculative data as confirmed.

**Comment by ouinouin** [comment]
the count in france based on this dataset leads me to calculate through grafana a 1500 death per day...
are the data still unconsolidated (some countries are showing cumulative data and some are showing daily data on the datasets .

**Comment by WildH0g** [comment]
Please, please, please fix France 🙏

**Comment by macialek** [comment]
Here is another source with proper France data: https://data.europa.eu/euodp/en/data/dataset/covid-19-coronavirus-data

**Comment by GerardoRTobar** [comment]
So what is the conclusion? Is the data wrong or not? As I understood, it is due to that French Government decided to count supicious cases as confirmed and then they turned out negative, isn't it?

Is the data manager going to fix it or not? @CSSEGISandData

**Comment by judepayne** [comment]
Does anyone have the correct numbers to hand for France for the 13th and 14th? I can put a patch into my copy of the jhu data. thank you

**Comment by yetzt** [comment]
@judepayne https://dashboard.covid19.data.gouv.fr/ — you can go to previous dates with the arrows top left.

**Comment by ivandebono** [comment]
Can anyone point me to a link for the data used here: https://dashboard.covid19.data.gouv.fr ?
Is there a csv available somewhere?

**Comment by boogheta** [comment]
@ivandebono THe dashboard uses the data from the following repository which agregates various official french sources: https://github.com/opencovid19-fr/data
The CSV file is there: https://raw.githubusercontent.com/opencovid19-fr/data/master/dist/chiffres-cles.csv
You'll see there are plenty different formats and granularity.
I personally do the [following filterings](https://github.com/boogheta/coronavirus-countries/blob/master/bin/export-main-countries.py#L310-L344) to account this data in my comparison dashboard: http://boogheta.github.io/coronavirus-countries/#country=France

**Comment by ivandebono** [comment]
@boogheta Super. Thank you so much.

**Comment by GerardoRTobar** [comment]
Is something going to be done about this???????????????????????

@CSSEGISandData At least can you reply back saying that we will be ignored or that we will be listened to

**Comment by judepayne** [comment]
@yetzt thankyou. So, when I reconcile back to that site, the numbers stretching back quite a while are substantially different to jhu...
@boogheta 

|jhu date| jhu | gouv.fr |
|----|----|----|
|15-Apr|134,582|106,206|
|14-Apr|131,361|103,573|
|13-Apr|137,875|98,076|
|12-Apr|133,670|95,403|
|11-Apr|130,727|93,790|
|10-Apr|125,931|90,767|
|9-Apr|118,781|86,334|
|8-Apr|113,959|82,048|
|7-Apr|110,065|78,176|
|6-Apr|98,963|74,390|
|5-Apr|93,773|70,478|
|4-Apr|90,848|68,605|
|3-Apr|65,202|64,338|
|2-Apr|59,929|59,105|

There was a big jump in the jhu cases number for France on the 4th April. At the time, I researched that spike and found somewhere (can't remember the source) that France had decided to include suspicious but unconfirmed deaths mainly from care homes.

Which is the correct (official) source? https://dashboard.covid19.data.gouv.fr/  pointed to by @yetzt or https://github.com/opencovid19-fr/data pointed to by @boogheta and used by jhu?

**Comment by boogheta** [comment]
@judepayne: as I was saying, the data repo I pointed IS the data source used by the official(ized) dashboard (it used to be a private initiative from people working for the government, which was then officially endorsed). You can confirm it by checking the source code here: https://github.com/etalab/covid19-dashboard/blob/master/prepare-data.js#L9
